config/config/sc-config.yaml

```diff
@@ -290,6 +302,126 @@ dex:
     # For more details, see https://elastisys.com/elastisys-engineering-how-to-use-dex-with-google-accounts-to-manage-access-in-kubernetes/
     # SASecretName: set-me
 
+thanos:
+  query:
+    # Enables the query/query-frontend components
+    enabled: true
+
+    # Number of query pods
+    # They are infinitly scalable
+    replicaCount: 1
+
+    resources:
+      requests:
+        cpu: 50m
+        memory: 60Mi
+      limits:
+        cpu: 500m
+        memory: 1Gi
+
+  queryFrontend:
+    resources:
+      requests:
+        cpu: 50m
+        memory: 40Mi
+      limits:
+        cpu: 300m
+        memory: 100Mi
+
+  receiveDistributor:
+    resources:
+      requests:
+        cpu: 100m
+        memory: 200Mi
+      limits:
+        cpu: 500m
+        memory: 200Mi
+
+  ruler:
+    # Enables the ruler component
+    # This is used as an alertmanager for long term data.
+    # It's recommended to not use this if not strictly necessary
+    enabled: false
+    resources:
+      requests:
+        cpu: 50m
+        memory: 40Mi
+      limits:
+        cpu: 300m
+        memory: 100Mi
+
+  receiver:
+    persistence:
+      enabled: true
+      size: 8Gi
+
+    # Sets the mode of operation for the receiver component
+    # dual-mode: Multiple instances of the receiver. For redundancy
+    # standalone: Single instance of receiver.
+    mode: dual-mode
+
+    # Retention for the metrics in the receiver
+    tsdbRetention: 15d
+
+    # Number of receiver instances
+    replicaCount: 2
+
+    # How many nodes that a time-series needs to be written to for a write to be considered successful.
+    # It needs to write to (REPLICATION_FACTOR + 1)/2 nodes for success.
+    # For more info: https://thanos.io/v0.24/proposals-done/201812-thanos-remote-receive.md/#replication
+    replicationFactor: 1
```

Comment:
So only  1 as default.
Any benefits making this 2?

Reply:
If we make this 2 then we `(REPLICATION_FACTOR + 1)/2` == `(2 + 1) / 2 = 1.5` aka needs to write to 1 node so that will be the same.
But if we set it to 3 so that it'll be `(3+1)/2 = 2` then the distributor will need to write to both receivers before a write is successful. I don't think we want this since half the point of having 2 is that 1 should be able to be down and it should still work.